src/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -380,6 +343,82 @@ else if (callVariantContextType == INDEL) {
+                                           final String truthSample,
+                                           final Optional<VariantContext> callContext,
+                                           final String callSample,
+                                           final int minGq, final int minDp) {
+        return classifyVariants(truthContext, truthSample, callContext, callSample, Optional.empty(), Optional.empty(), minGq, minDp);
+    }
+
+    /**
+     * Attempts to determine the concordance state given the truth and all variant context.  Optionally increments the genotype concordance
+     * counts for SNPs and Indels respectively.  This will ignore cases where an indel was found the truth and SNP  was found  in the call,
+     * and vice versa.  Typically, we fail to classify Mixed, Symbolic variants, or MNPs.
+     *
+     * @param truthContext A variant context representing truth
+     * @param truthSample The name of the truth sample
+     * @param callContext A variant context representing the call
+     * @param callSample The name of the call sample
+     * @param snpCounter optionally a place to increment the counts for SNP truth/call states
+     * @param indelCounter optionally a place to increment the counts for SNP truth/call states
+     * @param minGq Threshold for filtering by genotype attribute GQ
+     * @param minDp Threshold for filtering by genotype attribute DP
+     * @return true if the concordance state could be classified.
+     */
+    public static boolean classifyVariants(final Optional<VariantContext> truthContext,
+                                           final String truthSample,
+                                           final Optional<VariantContext> callContext,
+                                           final String callSample,
+                                           final Optional<GenotypeConcordanceCounts> snpCounter,
+                                           final Optional<GenotypeConcordanceCounts> indelCounter,
+                                           final int minGq, final int minDp) {
+        final VariantContext.Type truthVariantContextType = truthContext.map(VariantContext::getType).orElse(NO_VARIATION);
+        final VariantContext.Type callVariantContextType  = callContext.map(VariantContext::getType).orElse(NO_VARIATION);
+
+        // A flag to keep track of whether we have been able to successfully classify the Truth/Call States.
+        // Unclassified include MIXED/MNP/Symbolic...
+        final TruthAndCallStates truthAndCallStates = determineState(truthContext.orElse(null), truthSample, callContext.orElse(null), callSample, minGq, minDp);
+        if (truthVariantContextType == SNP) {
+            if ((callVariantContextType == SNP) || (callVariantContextType == MIXED) || (callVariantContextType == NO_VARIATION)) {
+                // Note.  If truth is SNP and call is MIXED, the event will be logged in the indelCounter, with row = MIXED
```

Comment:
These were wrong before this PR, so I will fix them to what I think they *should* be.